demo/audio_debugger/run.py

```diff
@@ -0,0 +1,18 @@
+import gradio as gr
+import subprocess
+import os
+
+audio_file = os.path.join(os.path.dirname(__file__), "cantina.wav")
```

Comment:
Why do we have these `os.path.join(os.path.dirname())` in all our demos?  You should be able to just put the filename, and users who look at demos for examples will think that they have to do this.

Reply:
Yeah that's a good point. But you need this to be able to run the demo files from any location